glide.yaml

```diff
@@ -1,51 +1,81 @@
 package: github.com/projectcalico/libcalico-go
 import:
+# Pin to v3.3.8 since updates on master use a breaking version of grpc.
+# The version of golang.org/x/sys that other dependencies use does not have
+# all the unix methods that the updated version of grpc requires.
 - package: github.com/coreos/etcd
-  version: ^3.3.0
+  version: v3.3.8
   subpackages:
   - client
   - clientv3
   - mvcc/mvccpb
   - pkg/transport
+- package: github.com/coreos/go-semver
+  subpackages:
+  - semver
 - package: github.com/kelseyhightower/envconfig
-  version: ^1.3.0
 - package: github.com/onsi/ginkgo
-  version: ^1.4.0
+- package: github.com/onsi/gomega
+  subpackages:
+  - types
 - package: github.com/projectcalico/go-yaml-wrapper
 - package: github.com/prometheus/client_golang
-  version: ^0.9.0-pre1
   subpackages:
   - prometheus
+# 1.1.0+ needed since our code expects the v4 uuid creation function to return a single value.
 - package: github.com/satori/go.uuid
   version: ^1.1.0
 - package: github.com/sirupsen/logrus
-  version: ^1.0.3
```

Comment:
I believe we had pinned this because later versions were not compatible with our log formatting code.

Maybe it's OK if all importing codebases pin this, but worth considering.

Reply:
I've restored the logrus pin and added a comment